src/Commands.pm

```diff
@@ -7267,4 +7268,48 @@ sub cmdSearchStore {
 			"searchstore buy [view|end|<item #> [<amount>]] : Buys from a store using Universal Catalog Gold\n");
 }
 
+sub cmdRevive {
+	my ($cmd, $args) = @_;
+	
+	if (!$net || $net->getState() != Network::IN_GAME) {
+		error TF("You must be logged in the game to use this command '%s'\n", $cmd);
+		return;
+	}
+	
+	if (!$char->{dead}) {
+		error TF("You must be dead to use this command '%s'\n", $cmd);
+		return;
+	}
+	
+	my @args = parseArgs($args);
+	my $item;
+	
+	if (scalar @args == 1) {
+		# User passed an item nameID
+		if ($args[0] =~ /^\d+$/) {
+			$item = $char->inventory->getByNameID($args[0]);
+		}
+		# User passed an item name
+		elsif ($args[0] ne "force") {
+			$item = $char->inventory->getByName($args[0]);
+		}
+	} elsif (scalar @args == 0) {
+		# Try to find Token Of Siegfried
+		$item = $char->inventory->getByNameID(7621);
+	} else {
+		error T("Error in 'revive' command (incorrect syntax)\n".
+				"revive [force|<item name>|<item ID>]\n");
+		return;
+	}
+	
+	if (!$item && $args[0] ne "force") {
```

Comment:
the way it is made
is impossible to declare a different item to revive and the "force" arg

if user type something like `revive 1234 force`, and item does not exist, "force" arg will not be respected

Reply:
The point of force is NOT specifying an item.
This item check exists solely to avoid trying when it's impossible - something the client would never do.
The item is not sent with the packet

Reply:
oh i understand. 
well, then for me the code are perfect :smile: